av/include/ignition/common/AudioDecoder.hh

```diff
@@ -22,25 +22,19 @@
 #include <memory>
```

Comment:
I think this header can be removed now?

Reply:
same for the other `<memory>` includes in this PR

Reply:
Ah, good catch, thanks.